### PR TITLE
[EOC 127] Display cohorts on separate route and add link in menu

### PR DESCRIPTION
**Pull request by alemikolo**
Dashboard view:
download

Cohort view:
chrome-capture

### CODE REVIEW

File: client/src/modules/dashboard/index.jsx [hunk 1]
```diff
@@ -2,110 +2,125 @@ import React, { Component, Fragment } from 'react';
...
 import GridList from 'common/components/GridList';
 import { CardColorType } from 'common/components/CardItem';
-import FormDialog, { FormDialogContext } from 'common/components/FormDialog';
+import FormDialog from 'common/components/FormDialog';
 import { Routes } from 'common/constants/enums';
 
 class Dashboard extends Component {
   state = {
-    dialogContext: ''
+    showArchivedLists: false,
```
Comment (adamklepacz) [inline client/src/modules/dashboard/index.jsx]: consider renaming to areArchivedListsVisible to match current pattern

File: client/src/modules/dashboard/index.jsx [hunk 2]
```diff
@@ -2,110 +2,125 @@ import React, { Component, Fragment } from 'react';
...
-        break;
-      case FormDialogContext.CREATE_LIST:
-        createList(data);
-        break;
-      default:
-        break;
-    }
-
-    this.hideDialog();
+  handleArchivedListsVisibility = () => {
```
Comment (adamklepacz) [inline client/src/modules/dashboard/index.jsx]: unecessary braces

File: server/controllers/list.js [hunk 3]
```diff
@@ -66,17 +66,22 @@ const getListsMetaData = (req, resp) => {
     user: { _id: userId }
   } = req;
 
+  const query = {
+    $or: [
+      { ownerIds: req.user._id },
```
Comment (adamklepacz) [inline server/controllers/list.js]: You can use userId variable here

**Comment by adamklepacz** [comment]
End_of_Coffee_-_track_your_office_inventory
Archived cohorts lists and private lists are not separated in this Archived Lists block.
was this desired by the client?

**Comment by alemikolo** [comment]
> End_of_Coffee_-_track_your_office_inventory
> Archived cohorts lists and private lists are not separated in this Archived Lists block.
> was this desired by the client?

I had a note where only three positions are mentioned: private lists, cohort's lists and archived lists. So let's wait and ask Łukasz.

**Pull request merged**